CMakeLists.txt

```diff
@@ -193,6 +193,22 @@ elseif (CMAKE_C_COMPILER_ID STREQUAL "GNU")
     else()
         message(STATUS "Setting GNU C compiler options with c11 and Posix")
         set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11 -pedantic -Wall -Wextra")
+        if (GCC_VERSION VERSION_GREATER_EQUAL 8.0)
+            # Disable some GCC checks:
+            #
+            # -Wstringop-truncation:
+            # strncpy into char arrays in FlatBuffer structs, these are
+            # zero-paddded but not zero terminated
+            #
+            # -Wno-format-overflow:
+            # GCC 9 will warn if you try to printf a NULL.
+            #
```

Comment:
We are not trying to printf a NULL and doing so would be bad. It is just that gcc mistakenly think that we do.

```
 # -Wno-format-overflow:
 # GCC 9 warns on mistakenly assumed NULL character when
 # printing from a required FlatBuffer string field.
```